### ISSUE TITLE
Image path in images.tex incorrect

### ISSUE THREAD

**Issue by ahogen** [latex2html/latex2html#1]
I don't know if this is the root cause of the image conversion error I keep getting or not, but it does seem incorrect. I am using LaTeX2HTML Version 2016 **(1.71)**

The error I keep getting when issuing `latex2html <filename>.tex` is...

```
pstoimg: Error: "/usr/bin/pnmcrop -verbose  < /tmp/l2h8747/p8764.pnm | /usr/bin/ppmquant -floyd 256 | /usr/bin/pnmtopng -interlace -trans '#d9d9d9'  > img1.png" failed: 

Error while converting image
```

While trying to debug this issue, I found out that the path written to the `images.tex` file is incorrect, to my understanding. Because images.tex is in a new directory (the output dir), it is one level deeper than the path to the image. Manually adding an additional `../` to the image path fixes the error and `images.tex` will then compile with, say, pdflatex, correctly.

Example:

```
% images.tex
\begin{document}

% (stuff and things here)

% !!! IMAGES START HERE !!!

\stepcounter{section}
{\newpage\clearpage
\lthtmlpictureA{tex2html_wrap181}%
\includegraphics[width=0.7\linewidth]{"../img/phys-design/rough-physical-design"}%
% ^^ This path should be "../../img/phys-design/rough-physical-design" to compile correctly with pdflatex
\lthtmlpictureZ
\lthtmlcheckvsize\clearpage}

\stepcounter{section}

\end{document}
```

**Comment by ahogen** [comment]
I re-`make`ed and installed with setting `$prefs{'PIPES'} = 0;` on line 371 of prefs.pm.

When I re-issue the `latex2html <filename>.tex` command, I get the following errors when it attempts to convert the same image.

```
Converting image #1
pstoimg: Error: "/usr/bin/pnmcrop -verbose   < /tmp/l2h11539/p11555.pnm > /tmp/l2h11539/p11555.t00" failed: Illegal seek

Error while converting image

Error: Cannot read 'img1.png': No such file or directory

Doing section links ......
Doing table of contents ..........
```

So I get failures on all three steps of the conversion.

Any thoughts? Thanks!

**Comment by danielgildea** [comment]
should be fixed by commit [9477ad25699716308dabaf8c0e9ea115d867a4a3]

see also http://tug.org/pipermail/latex2html/1999-June/000209.html
